build/build.sh

```diff
@@ -139,18 +139,30 @@ dascandy-contracts-trunk)
     LLVM_ENABLE_RUNTIMES+=";libunwind"
     ;;
 rocm-*)
+    ROCM_VERSION=9999
```

Comment:
what's the significance of 9999 here? If you need to set this to something, maybe empty is better than this? or `declare` it? (not a huge bash expert here)

Reply:
To save on back and forth I'm going to merge this with tweaks to fix this

Reply:
I looked harder and...I think this just need a comment. Will merge :) thanks!

Reply:
9999 is for the default situation, including trunk, where we assume llvm includes device libs and comgr, in contrary to the old release branches which do not include device libs and comgr.